docs/source/howto/include/scripts/performance_benchmark_base.py

```diff
@@ -9,7 +9,7 @@
 
 @click.command()
 @options.CODE(required=False, help='A code that can run the ``ArithmeticAddCalculation``, for example bash.')
-@click.option('-n', 'number', type=int, default=100, show_default=True, help='The number of processes to submit.')
+@click.option('-n', 'number', type=int, default=10, show_default=True, help='The number of processes to submit.')
```

Comment:
Now see that I forgot the help string when making the default to run instead of submitting
```suggestion
@click.option('-n', 'number', type=int, default=10, show_default=True, help='The number of processes to launch.')
```